### PR TITLE
Add `osu! Malaysia @ AniManGaki 2023` news post

### PR DESCRIPTION
**Pull request by Hiviexd**
PRing on behalf of [Jerry](https://osu.ppy.sh/users/605973)

### CODE REVIEW

**Review by 0x84f — APPROVED** [review]
short and straight to the point, I think it's fine

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+layout: post
+title: "osu! Malaysia @ AniManGaki 2023"
+date: 2023-08-22 08:00:00 +0000
+---
+
+The osu! Malaysia community proudly presents its first community-run space happening at [AniManGaki 2023](https://www.animangaki.com/)!
+
+![](/wiki/shared/news/2023-08-22-osu-malaysia-animangaki/banner.jpg)
+
+The event will be happening at the Mines International Exhibition & Convention Centre Malaysia (*MIECC*) from 25th to 27th August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:11]: https://miecc.mines.com.my/about/

.. although whether "mines" is capitalised seems inconsistent between sources <.<

```suggestion
The event will be happening at the MINES International Exhibition & Convention Centre (*MIECC*) in Malaysia from 25 to 27 August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
```

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## LAN Tournament
+
+The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times (UTC+8):
+
+- **Friday, 3:00PM:** osu!catch
+- **Saturday, 12:00PM:** osu!taiko
+- **Saturday, 3:00PM:** osu!mania
+- **Sunday, 2:00PM:** osu!
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:20]:
```suggestion
| Game mode | Timestamp (UTC+08) |
| :-- | :-- |
| osu!catch | Friday, 25 August 2023 (15:00) |
| osu!taiko | Saturday, 26 August 2023 (12:00) |
| osu!mania | Saturday, 26 August 2023 (15:00) |
| osu! | Sunday, 27 August 2023 (14:00) |
```

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The osu! Malaysia community proudly presents its first community-run space happening at [AniManGaki 2023](https://www.animangaki.com/)!
+
+![](/wiki/shared/news/2023-08-22-osu-malaysia-animangaki/banner.jpg)
+
+The event will be happening at the Mines International Exhibition & Convention Centre Malaysia (*MIECC*) from 25th to 27th August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
+
+## LAN Tournament
+
+The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times (UTC+8):
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:15]:
```suggestion
The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times:
```

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- **Saturday, 3:00PM:** osu!mania
+- **Sunday, 2:00PM:** osu!
+
+Prizes for the winners of each mode includes exclusive booth merchandises and physical trophies!
+
+## Booth Activities
+
+Besides the exciting tournament matches, several other activities such as daily offline leaderboards, score challenges, contests, giveaways and more will be happening at the booth for everyone to enjoy, allowing for many new memories to be made together!
+
+For all the latest news on the booth happenings throughout the event, do visit the official [o!MY@AMG2023 Discord Server](https://discord.gg/ZKKWb8jK2B) or follow [@osumalaysia on Twitter](https://twitter.com/osumalaysia).
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:28]:
```suggestion
For all the latest news on the booth happenings throughout the event, do visit the official [osu! Malaysia @ AMG2023 Discord server](https://discord.gg/ZKKWb8jK2B) or follow [@osumalaysia on Twitter](https://twitter.com/osumalaysia).
```

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+date: 2023-08-22 08:00:00 +0000
+---
+
+The osu! Malaysia community proudly presents its first community-run space happening at [AniManGaki 2023](https://www.animangaki.com/)!
+
+![](/wiki/shared/news/2023-08-22-osu-malaysia-animangaki/banner.jpg)
+
+The event will be happening at the Mines International Exhibition & Convention Centre Malaysia (*MIECC*) from 25th to 27th August 2023, providing an opportunity for osu! players all around the country to meet up and mingle!
+
+## LAN Tournament
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:13]:
```suggestion
## LAN tournament
```

File: news/2023/2023-08-22-osu-malaysia-animangaki.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+The main highlight of the event includes the final bracket stages of the **osu! Malaysian LAN Tournament** for all four game modes, with each of the top 4 competing live at the following times (UTC+8):
+
+- **Friday, 3:00PM:** osu!catch
+- **Saturday, 12:00PM:** osu!taiko
+- **Saturday, 3:00PM:** osu!mania
+- **Sunday, 2:00PM:** osu!
+
+Prizes for the winners of each mode includes exclusive booth merchandises and physical trophies!
+
+## Booth Activities
```
Comment (Walavouchey) [inline news/2023/2023-08-22-osu-malaysia-animangaki.md:24]:
```suggestion
## Booth activities
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**